### PR TITLE
Add OpenMP on the top

### PR DESCRIPTION
**Pull request by ShalokShalom**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,9 @@
...
 [![Build Status](https://travis-ci.com/illuhad/hipSYCL.svg?branch=master)](https://travis-ci.com/illuhad/hipSYCL)
 
-The goal of the hipSYCL project is to develop a SYCL 1.2.1 implementation that builds upon NVIDIA CUDA/AMD HIP. hipSYCL consists of a source-to-source transformation toolchain to automatically convert SYCL code into CUDA/HIP code behind the scenes, and an implementation of a SYCL runtime that runs on top of CUDA/HIP. The actual compilation step is then carried out with the regular NVIDIA and AMD compilers for CUDA and ROCm, which allows hipSYCL applications to make use of all the latest CUDA and ROCm features (e.g. intrinsics). This approach also guarantees compatibility of hipSYCL applications with established and well-supported vendor tools (e.g. profilers) and libraries in the NVIDIA and AMD ecosystems.
+The goal of the hipSYCL project is to develop a SYCL 1.2.1 implementation that builds upon NVIDIA CUDA, AMD HIP and OpenMP. hipSYCL consists of a source-to-source transformation toolchain to automatically convert SYCL code into CUDA/HIP/OpenMP code behind the scenes, and an implementation of a SYCL runtime that runs on top of them.
```
Comment (illuhad) [inline README.md]: The source-to-source transformation doesn't really convert code into OpenMP. In order to have an OpenMP compiler accept SYCL code, no toolchain changes are necessary. This if for example how triSYCL works :)

**Comment by illuhad** [comment]
Thank you very much for this PR! However, I feel that OpenMP in hipSYCL is more of an implementation detail - there's not a single OpenMP line of code in hipSYCL itself. It's only used internally by the hipCPU library, which exposes a HIP interface on top of OpenMP. Also, CPU support in hipSYCL is pretty unoptimized and mostly only used for debugging. So, from that perspective, I would prefer not promoting it right in the header for now.
But I'm very open to adding a paragraph that mentions that we have a CPU backend on top of OpenMP :)

**Comment by ShalokShalom** [comment]
You have?

**Comment by illuhad** [comment]
I'm sorry, I'm not quite sure what you mean. Could you please elaborate? Do you mean the name of the CPU backend? It's powered by the [hipCPU](https://github.com/illuhad/hipCPU) library.

**Comment by ShalokShalom** [comment]
Could we refer to hipGPU instead?

**Pull request closed without merging**